index.js

```diff
@@ -66,53 +82,44 @@ class KoaWebMonetization {
     this.buckets.set(id, balance - price)
   }
 
-  paid ({ price, awaitBalance = false }) {
-    return async (ctx, next) => {
-      const id = ctx.params.id
-      if (!id) {
-        return ctx.throw(400, 'ctx.params.id must be defined')
-      }
-
-      const _price = (typeof price === 'function')
-        ? Number(price(ctx))
-        : Number(price)
-
-      if (awaitBalance) {
-        await this.awaitBalance(id, _price)
-      }
+  async receive (ctx) {
```

Comment:
I think we should define a method `receiver` that returns this `receive(ctx)` function, so the user doesn't have to call `.bind` themself. It's a fairly common pattern for Koa related modules to use functions that return middlewares.